protocol.md

```diff
@@ -182,6 +182,29 @@ The `Tus-Max-Size` response header MUST be a non-negative integer indicating the
 allowed size of an entire upload in bytes. The Server SHOULD set this header if
 there is a known hard limit.
 
+#### Tus-Min-Chunk-Size
+
+The `Tus-Min-Chunk-Size` response header MUST be a non-negative integer indicating the
+recommended minimum size of a single chunk uploaded in a single `PATCH` request in bytes.
```

Comment:
I would place "in bytes" more to the front: indicating the recommended minimum size in bytes of a single chunk... Remove 'in bytes' at the end.